include/ideep/operators/conv.hpp

```diff
@@ -1,20 +1,55 @@
 #ifndef IDEEP_OPERATORS_CONV_HPP
 #define IDEEP_OPERATORS_CONV_HPP
-
 namespace ideep {
 
+struct convolution_forward_quant_params {
+  convolution_forward_quant_params() {}
+
+  convolution_forward_quant_params(tensor&& src_zero_point)
+                                  : _src_zero_point(src_zero_point) {}
```

Comment:
```suggestion
                                  : _src_zero_point(std::move(src_zero_point)) {}
```

Reply:
Similar changes suggested to similar code below.

Reply:
Good catch of the missing move for rvalue conversion, updated related code in this file